packages/api/src/axelar-query/isomorphic.ts

```diff
@@ -319,4 +341,138 @@ export class AxelarQueryAPIClient extends RestService {
 
     return this.cachedChainConfig;
   }
+
+  /**
+   * Get a list of active chains.
+   * @returns an array of active chains
+   */
+  public async getActiveChains(): Promise<string[]> {
+    const axelarQueryRpcClient = (await createAxelarRPCQueryClient({
+      environment: this.env,
+    })) as AxelarQueryClientType;
```

Comment:
can we reuse [this type](https://github.com/axelarnetwork/axelarjs/blob/76f4a83a5c808753e7e0567c9862c5ca68b3a656/packages/cosmos/src/rpc/types.ts#L100)?